docs/source/license.rst

```diff
@@ -1,9 +1,6 @@
 License
 =======
 
```

Comment:
Not that I remember. It should be there with the rest of the license, could you add it back?